src/Cake.Common/Tools/OctopusDeploy/CreateReleaseSettings.cs

```diff
@@ -46,5 +46,15 @@ public sealed class CreateReleaseSettings : OctopusDeploySettings
         /// Gets or sets a value indicating whether the Ignore Existing flag.
         /// </summary>
         public bool IgnoreExisting { get; set; }
+
+        /// <summary>
+        /// Gets or sets the octopuschannel for the new release.
+        /// </summary>
+        public string Channel { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether channel rules should be ignored
```

Comment:
Wait, doesn't these tests cover this tool: https://github.com/cake-build/cake/blob/develop/src/Cake.Common.Tests/Unit/Tools/OctopusDeploy/OctoCreateReleaseTests.cs

On phone just now, so haven't checked properly.